Gemfile

```diff
@@ -5,5 +5,6 @@ gem "insist"
 gem "stud"
 gem "fpm"
 gem "pleaserun"
+gem "jls-lumberjack", :github => "elastic/ruby-lumberjack"
```

Comment:
I don't think we need this change, do we? The gem name isn't changing (yet), and if it does, we'll still publish it normally.